gramps/gen/filters/rules/event/__init__.py

```diff
@@ -71,4 +72,5 @@
     HasTag,
     MatchesPlaceFilter,
     HasDayOfWeek,
+    HasEventBase,
```

Comment:
How is `HasEventBase` and `HasSourceBase` related to this PR?  I can't see them referenced anywhere.

Reply:
The 'Define filter' feature does not work without it.

Event filtering uses the rule class HasEventBase and 'Define filter' will store that class name in custom_filters.xml. For example:

  <object type="Event">
    <filter name="test" function="and" comment="Created by Filter gramplet on 2024-05-12">
      <rule class="HasEventBase" use_regex="False" use_case="False">
        ...
      </rule>
  </object>

However, when custom_filters.xml is read and interpreted then the corresponding rule is silently ignored because the class is not imported in gramps.gen.filters.rules.event.__init__.py. There is only this message on the console:

    ERROR: Filter rule 'HasEventBase' in filter 'test' not found!

My solution is to import the class in the __init__.py file. At first i thought it also needed to be added in 'editor_rule_list' but that does not seem to be necessary.

Reply:
I don't see any warnings or errors if I comment these lines out.

The `HasEventBase` is used as the "Events with parameters" rule.  It works if I click the 'Define filter' button in the sidebar, or if I invoke the filter editor from the menu.